src/js/modules/load-breeds-home.js

```diff
@@ -0,0 +1,22 @@
+import getBreeds from './GET-breeds-home';
+import likeIcon from '../../img/heart-icon.png';
+
+const loadBreeds = async () => {
+  const dogBreeds = await getBreeds();
+  const listContainer = document.querySelector('.list-container');
+  dogBreeds.forEach((e) => {
```

Comment:
This loop is most likely where the resource-intensive computation happens. You could limit it to a reduced number of elements of the list.

Reply:
The issue is that it takes too long to load (several minutes on my computer). For now, you could do something like displaying the first few elements of the list to speed it up. Otherwise, I won't be able to continue developing with this change added.

Reply:
Done ! :+1: